microsoft-edge/webview2/release-notes.md

```diff
@@ -35,18 +35,23 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
 To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test upcoming APIs and features](how-to/set-preview-channel.md).
 
 
+<!-- ====================================================================== -->
+## Platforms and languages covered
+
+Generally, release notes apply to both Win32/C++ and .NET/C#.  In some cases, a release note specifies a particular platform that it applies to.  The [Win32/C++ API Reference](/microsoft-edge/webview2/reference/win32/icorewebview2) is parallel with the [.NET/C# API Reference](/dotnet/api/microsoft.web.webview2.core.corewebview2).

```

Comment:
IIUC release notes can apply to all Win32, .NET, and WinRT. Win32 `ICoreWebView2` and `ICoreWebView2_N` interfaces are roughly parallel to .NET and/or WinRT CoreWebView2 class.